Works & about me section

VelzckC0D3 opened the pull request:
# My Portfolio

# What I have submitted
- The `#Works` section trying to make it as similar as I could (as it's requested by Microverse)
- The `#AboutMe` section where you are able to see almost all the website finished.
- I optimized some CSS code and the font loading with HTML
- Netlify live demo updated! [Click here to access it!](https://velzckcode.netlify.app/)
# ❗♻️ New update | requested changes ♻️ ❗ 
- As my first reviewer noticed I was having semantic errors on the `See more` buttons on the grid, I corrected it for all 7 buttons!   
- Also, corrected the semantic error of having 2 `h1` on the same webpage to optimize the SEO! 

# ⬇️ New Features! ⬇️
✅ A very nailed replica of the mobile version of [Microverse Portfolio Template #2](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=JqSXbNsYkocPTz9n-0)
✅ Added "Works" section where you can see that I implemented Grid on the layout in order to accomplish the requirements
![works](https://user-images.githubusercontent.com/92229666/219870253-dbf0ccfb-9344-43bc-aeea-6ee04b670b9c.gif)

✅ Added "About Me" section. All this design is really adaptative to almost all screen sizes, I'm excited to continue with it! 
![aboutMe](https://user-images.githubusercontent.com/92229666/219870256-a6eb2b6e-7b86-4ab1-97af-cbab8608cbda.gif)

✅ Included `:hover` animations for all the buttons on the design! 
![hoverIcons](https://user-images.githubusercontent.com/92229666/219871154-48a96275-73c4-42e3-a4b0-5651c5600ef1.gif)

# ✨ Thank you for your review! ✨
![ThumbsUpMetalGearGIF](https://user-images.githubusercontent.com/92229666/219876308-701482ad-5bed-4d27-a125-7d8ec61b686b.gif)

VelzckC0D3 committed:
Works (Primary Card)

On this commit, I implemented properly the first card container with the images and the descriptions, it's fully adaptative

VelzckC0D3 committed:
Finished Works Section

Completely finished and polished details on the "Works" section, corrected all the Stylelint errors and optimized the font loading in the HTML file

VelzckC0D3 committed:
Works and AboutMe Complete

Added the "about me" section, also some hover animations to the buttons, then, optimized the HTML font loading

VelzckC0D3 committed:
Minor Button Changes

I updated some features on the grid button so it looks better.

brainconnect93 commented on index.html:
```diff
@@ -60,7 +56,241 @@ <h2 class="roboto">I'm a Software Developer</h2>
+                                    </ul>
+                                </div>
+                            </div>
+                            <div class="gridButton">see project</div>
```
- [x] Hi @VelzckC0D3, I must really commend your effort, in short, you have a very wonderful project, 👍 👍 I would have loved to approve this project at this your first submission but I really want to make this your project looks great semantically by requesting these changes.
  - Please I would suggest you make all the `See project` button wrapped inside of a `button` or an `a` tag for semantic reasons.
  E.g:
```HTML
<div class="gridButton"><a hef="#">see project</a></div> OR
<div class="gridButton"><button type="submit">see project</button></div>
```

VelzckC0D3 replied:
Hi, @brainconnect93, thank you so much for your words! I really, really appreciate that. I worked on it right away and solved it for all 7 buttons! 
![image](https://user-images.githubusercontent.com/92229666/219875688-1e316933-a214-4ec4-8b27-c22e1360f2b1.png)

brainconnect93 commented on index.html:
```diff
@@ -60,7 +56,241 @@ <h2 class="roboto">I'm a Software Developer</h2>
+                    </div>
+                    <div id="buttonCV">
+                        <a id="cvLink" class="inter" href="http://github.com/velzckC0D3" target="_blank" rel="noopener noreferrer">
+                            <h1>get my resume</h1>
```
- [x] Please kindly note that for best practice we can only use one `h1` in a webpage and this is for SEO (search engine optimization) and accessibility reasons. I would kindly suggest you remove this `h1` tag and leave your `Get my resume` button to be wrapped inside of an `anchor` tag alone. 👏 👏

VelzckC0D3 replied:
Got it! I toke out the ``Get my resume`` out of the H1 and now it's only inside the `anchor` I give it the properties with CSS and it's done! 
![image](https://user-images.githubusercontent.com/92229666/219875778-866bbf2a-6798-453d-9b7e-42eea95fbb38.png)

brainconnect93 reviewed (changes requested):
## STATUS: Changes Required :recycle:

### Hi @VelzckC0D3, 👋 

I must say this you have a fantastic work in your project, you have shown an high level of professionalism, and I will be sincere with you you are going places in this space as a software developer, KEEP IT UP!!! 🍾 🍾 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media.tenor.com/isiLRnxz3zwAAAAM/cat-driving-serious.gif" width="500px" height="300px">

### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

VelzckC0D3 committed:
Check Review Corrections

As requested to follow the best practices: I adjusted all the 7 "See project" buttons inside of an anchor, then, I toke out the "Get my resume" from the h1 and leave it inside of an anchor  to improve and follow the correct practices for an optimal SEO adjusting the properties on CSS

bobb-Rob commented on index.html:
```diff
@@ -60,7 +56,248 @@ <h2 class="roboto">I'm a Software Developer</h2>
                     </ul>
                 </div>
             </div>
+        </section>
```
A very beautiful project you here. Well done 👏🏾 🍾 

I see a few things that can further take you higher. 

- #### First Project Button
You currently have a colored border on this button. Following the Figma template, thiere should be not border. Consider this. 

Your Work | Figma Design
--- | ---
![image](https://user-images.githubusercontent.com/84156714/219877092-461a5329-0ccd-4799-add8-b4e790fb0436.png) | ![image](https://user-images.githubusercontent.com/84156714/219877162-a8fd5d31-7561-4095-9fef-2f1008a485b7.png)

- #### Commit message
Getting in the habit of creating quality commit messages makes using and collaborating with Git a lot easier. You will sure find very useful tips in [this article ](https://initialcommit.com/blog/git-commit-messages-best-practices) that will help you improve your commit messages.

VelzckC0D3 replied:
Thank you!! I appreciate your feedback, right now I'm going to eat that article !!

bobb-Rob reviewed (approved):
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @VelzckC0D3, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![](https://camo.githubusercontent.com/2c67944c14f5c74b9b607df18d73efb0c423dd148b3e10cb75b445a00220332e/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f5061536e555a573145657970756c734f77642f67697068792e676966)

## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
